delivery_dropoff_site/__manifest__.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+# Copyright (C) 2014 - Today: Akretion (http://www.akretion.com)
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
+# @author Aymeric Lecomte <aymeric.lecomte@akretion.com>
+# @author David BEAL <david.beal@akretion.com>
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Delivery Drop-off Sites',
+    'version': '10.0.1.0.0',
+    'author': 'Akretion,GRAP,Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'summary': "Send goods to sites in which customers come pick up package",
+    'category': 'Delivery',
+    'depends': [
+        'delivery',
+        'base_geolocalize',
+        'resource',
+    ],
+    'website': 'https://www.odoo-community.org',
```

Comment:
Can you please change website key to `https://github.com/OCA/delivery-carrier`

Reply:
indeed. (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules)
done.